chain/beacon/chain.go

```diff
@@ -41,12 +41,19 @@ type chainStore struct {
 func newChainStore(l log.Logger, cf *Config, cl net.ProtocolClient, c *cryptoStore, store chain.Store, t *ticker) *chainStore {
 	// we make sure the chain is increasing monotically
 	as := newAppendStore(store)
+
+	//
```

Comment:
small comments welcomed to indicate why we instantiate that store

Reply:
Done!